src/test/java/hw2/ex2/SecondScenarioTest.java

```diff
@@ -0,0 +1,278 @@
+package hw2.ex2;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static org.testng.Assert.*;
+
+public class SecondScenarioTest {
+
+    private WebDriver driver;
+    private SoftAssert sa;
+
+    @BeforeTest
+    public void setUp() {
+        WebDriverManager.chromedriver().setup();
+        this.driver = new ChromeDriver();
+        this.sa = new SoftAssert();
+        driver.manage().timeouts().
+                implicitlyWait(10000, TimeUnit.MILLISECONDS);
+        driver.manage().timeouts().
+                pageLoadTimeout(20000, TimeUnit.MILLISECONDS);
+        driver.manage().timeouts().
+                setScriptTimeout(30000, TimeUnit.MILLISECONDS);
+    }
```

Comment:
This is looks like a code duplication from the `FirstScenarioTest`.
Please avoid code duplication. Also browser window does not maximized

Reply:
Solved: I create abstract base class for those test classes and write there *@BeforeMethod* and *@AfterMethod* annotations to avoid code duplication